src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsEvolveObject.java

```diff
@@ -39,6 +43,12 @@ public void verifyEvolveCall() {
     verifyNoEvolveInSlot(AdsContent.PREFOOTER_RIGHT);
   }
 
+  public void verifyEvolveCallMercury() {
```

Comment:
What do you think about renaming the previous method to `verifyEvolveCallOasis()`?